Assignment3/Assignment3.py

```diff
@@ -0,0 +1,107 @@
+import sys
+
+from pyspark import StorageLevel
+from pyspark.ml.regression import RandomForestRegressor
+from pyspark.sql import SparkSession
+
+
+def main():
+    appName = "assignment3"
+    master = "local"
+    spark = (
+        SparkSession.builder.appName(appName)
+        .master(master)
+        .config(
+            "spark.jars",
+            "/Users/bitaetaati/PythonProjectTemplate/PythonProjectTemplate/mariadb-java-client-3.0.8.jar",
+        )
+        .getOrCreate()
+    )
+
+    sql1 = "select * from baseball.batter_counts"
+    database = "baseball"
```

Comment:
Don't put your password in plain text on the internet, you're going to want to use an environment variable.

Reply:
done